File: Arithmetic_CheckPrime.cpp
// Arithmetic.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include "CheckPrime.h"

#include <iostream>

using namespace std;


int main() {
	try {

		cout << "\n    Welcome to our demonstration sequels\n";
		cout << "Hope you enjoy (and follow) the lessons.\n\n";


		unsigned int start = 1, stop = 100;

		/*
		* Use the CheckPrime class.
		*/
		CheckPrime answer(98);
		cout << "\n\n" << answer.verifyPrime() << "\n";

	}	catch (exception& e) {
		cout << "\n" << e.what() << "\n";
	}
    return 0;
}

File: Arithmetic_CheckPrime.cs
using System;
using System.Collections.Generic;

namespace Arithmetic_CS
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Welcome to our demonstration sequels");
            Console.WriteLine("Hope you enjoy (and follow) the lessons.");
            Console.WriteLine("\r\n");

            /*
            * Test for the primeness of a number.
            */
            int test_guy = 97;
            CheckPrime prime_check = new CheckPrime(test_guy); // Is 97 is prime number?

            string result = "Prime State:\r\n";
            if (prime_check.verifyPrime())
            {
                result += test_guy + " is a prime number.";
            }
            else
            {
                result += test_guy + " is not a prime number.";
            }

            Console.WriteLine(result);

        }
    }
}
